src/main/java/gregtech/common/metatileentities/electric/MetaTileEntityFisher.java

```diff
@@ -60,7 +61,7 @@ protected ModularUI createUI(EntityPlayer entityPlayer) {
         int rowSize = (int) Math.sqrt(inventorySize);
 
         ModularUI.Builder builder = ModularUI.builder(GuiTextures.BACKGROUND, 176,
-                18 + 18 * rowSize + 94)
+                        18 + 18 * rowSize + 94)
```

Comment:
Can you revert this, I think it may conflict with the ModularUI work and is a simple merge conflict to avoid